sdk/tables/Azure.Data.Tables/src/TableServiceClient.cs

```diff
@@ -91,6 +92,30 @@ internal TableServiceClient(TableRestClient internalClient)
         protected TableServiceClient()
         { }
 
+        /// <summary>
+        /// Gets a <see cref="TableSasBuilder"/> instance scoped to the current account.
+        /// </summary>
+        /// <param name="permissions"><see cref="TableAccountSasPermissions"/> containing the allowed permissions.</param>
+        /// <param name="resourceTypes"><see cref="TableAccountSasResourceTypes"/> containing the accessible resource types.</param>
+        /// <param name="expiresOn">The time at which the shared access signature becomes invalid.</param>
+        /// <returns>An instance of <see cref="TableAccountSasBuilder"/>.</returns>
+        public virtual TableAccountSasBuilder GetSasBuilder(TableAccountSasPermissions permissions, TableAccountSasResourceTypes resourceTypes, DateTimeOffset expiresOn)
+        {
+            return new TableAccountSasBuilder(permissions, resourceTypes, expiresOn) { Version = _version };
```

Comment:
The Storage team has decided to only support signing SAS tokens with the latest version (i.e., the constant value on your `TableSasQueryParameters` type) so we don't have to deal with multiple signing algorithms.